doc/source/cluster/cloud.rst

```diff
@@ -184,11 +184,55 @@ Private Cluster (List of nodes)
 
 The most preferable way to run a Ray cluster on a private cluster of hosts is via the Ray Cluster Launcher.
```

Comment:
Can we add a definition of a "Private Cluster" I don't want users to get confused about what this section actually represents and whether or not they need to follow it in the general case.

Please add a couple of sentences with the "why would I want a private cluster?" "when would I use this?"